pkgs/applications/misc/mapproxy/default.nix

```
@@ -38,6 +38,7 @@ buildPythonApplication rec {
     substituteInPlace mapproxy/util/ext/serving.py --replace "args = [sys.executable] + sys.argv" "args = sys.argv"
   '';
   propagatedBuildInputs = [
+    boto3
```

do you mind adding a note as to why this is necessary?
```suggestion
    boto3 # needed for caches service
```

---

Done.